api/nodes/views.py

```diff
@@ -1142,47 +1152,55 @@ def postprocess_query_param(self, key, field_name, operation):
             ]
 
     def get_default_queryset(self):
-        files_list = self.fetch_from_waterbutler()
-
-        if isinstance(files_list, list):
-            provider = self.kwargs[self.provider_lookup_url_kwarg]
-            # Resolve to a provider-specific subclass, so that
-            # trashed file nodes are filtered out automatically
-            ConcreteFileNode = BaseFileNode.resolve_class(provider, BaseFileNode.ANY)
-            file_ids = [f.id for f in self.bulk_get_file_nodes_from_wb_resp(files_list)]
-            return ConcreteFileNode.objects.filter(id__in=file_ids)
-
-        if isinstance(files_list, list) or not isinstance(files_list, Folder):
-            # We should not have gotten a file here
-            raise NotFound
+        if self.kwargs.get('node_id'):
```

Comment:
try just using `get_resource`, as PreprintFilesList already overrides this (WaterButlerMixin just calls get_node)